Resources/views/Form/fields.html.twig

```diff
@@ -384,8 +385,10 @@
       <div {{ block('widget_container_attributes') }}>
         {{ form_errors(form.date) }}
         {{ form_errors(form.time) }}
-        <div class="col">{{ form_widget(form.date, {'dont_render_row': true, 'attr': {'class': attr.widget_class|default('')}, 'horizontal_input_wrapper_class': horizontal_input_wrapper_class|default('col-sm-3')}) }}</div>
-        <div class="col">{{ form_widget(form.time, {'dont_render_row': true, 'attr': {'class': attr.widget_class|default('')}, 'horizontal_input_wrapper_class': horizontal_input_wrapper_class|default('col-sm-2')}) }}</div>
+        <div
+          class="col">{{ form_widget(form.date, {'dont_render_row': true, 'attr': {'class': attr.widget_class|default('')}, 'horizontal_input_wrapper_class': horizontal_input_wrapper_class|default('col-sm-3')}) }}</div>
+        <div
```

Comment:
Ik vind een enter na `<div class="col">` hier logischer, maar dit is wss automatisch?

Reply:
Fixed in bde5c21f40896f658a8c64f84d1e7f4fc91ad1ba